src/azul/plugins/metadata/hca/transform.py

```diff
@@ -80,6 +83,43 @@
 pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
 
 
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: JSON) -> str:
+        try:
+            value, unit = value['value'], value['unit']
+        except TypeError:  # Used for translating None
+            value, unit = [None, None]
+        except KeyError:
+            require({'value', 'unit'} == value.keys(),
```

Comment:
```suggestion
            require(value.keys() == {'value', 'unit'},
```

Same for all other comparisons. The natural order of operands to `==` is `variable_or_expression == literal` not the other way around.